package.json

```diff
@@ -35,6 +35,15 @@
       "js",
       "vue"
     ],
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "**/*.{js,vue}",
+      "!**/node_modules/**"
+    ],
+    "coverageReporters": [
```

Comment:
I think this should be omitted as jest will generate text, html and lcov by default

Reply:
Sure, this is just an example, but I can remove it.

Reply:
After upgrading jest and other dependencies, it doesn't seem to collect coverage from vue files